plugins/DBStats/templates/index.twig

```diff
@@ -42,7 +42,7 @@
     <tbody>
     <tr>
         <td>
-            <h2>{{ 'DBStats_TrackerTables'|translate }}</h2>
+            <h2 class="secondary">{{ 'DBStats_TrackerTables'|translate }}</h2>
```

Comment:
I'd rather avoid new generic classes to style headers manually like this. Especially since it seems to be used only in the DB statistics page. Maybe this should be a h3 if it's a "secondary" h2? Or if only those headers should be customized it could be using a selector on a HTML tag that contains them? Just throwing ideas